simapp/v2/app_config.go

```diff
@@ -181,7 +184,7 @@ var (
 					// OrderMigrations: []string{},
 					// TODO GasConfig was added to the config in runtimev2.  Where/how was it set in v1?
 					GasConfig: &runtimev2.GasConfig{
-						ValidateTxGasLimit: 100_000,
+						ValidateTxGasLimit: 10_000_000,
```

Comment:
should leave a comment that is this for testing only. Lots of teams copy our code without double checking it